src/mixins/FeeCollectable.sol

```diff
@@ -14,7 +13,7 @@ abstract contract FeeCollectable {
         uint16 feeAmount
     );
 
-    uint256 private constant BIPS = 10_000;
+    uint256 internal constant BIPS = 10_000;
```

Comment:
any reason for the change? just curious

Reply:
to avoid repeating it for contract that inherits the FeeCollectable contract so that BIPS is available